src/containers/SearchPage/README.md

```diff
@@ -60,7 +64,8 @@ if SearchMap's bounds have changed enough.
 
 ### Search filters
 
-See the [filters documentation](../../../docs/search-filters.md).
+See the
+[filters documentation](https://www.sharetribe.com/docs/cookbook-search/change-search-filters-in-ftw/).
```

Comment:
I assume this article will be updated before releasing this?

Reply:
Yes.